sphinx/ext/napoleon/docstring.py

```diff
@@ -600,6 +601,27 @@ def _parse_attribute_docstring(self) -> List[str]:
     def _parse_attributes_section(self, section: str) -> List[str]:
         lines = []
         for _name, _type, _desc in self._consume_fields():
+            # code adapted from autodoc.AttributeDocumenter:add_directive_header
+            if not _type and self._what in ('class', 'exception') and self._obj:
```

Comment:
I think we can use this feature to module-level variables also.

Reply:
What would be the value of `_what` for module-level variables ? Is it `module`, or another value not reflected in `GoogleDcostring` own docstring ?

Reply:
Just understood what you meant, activated it for `module` too